python/lsst/sims/maf/plots/plotHandler.py

```diff
@@ -566,10 +566,10 @@ def saveFig(self, fignum, outfileRoot, plotType, metricName, slicerName,
         fig = plt.figure(fignum)
         plotFile = outfileRoot + '_' + plotType + '.' + self.figformat
         if self.trimWhitespace:
-            fig.savefig(os.path.join(self.outDir, plotFile), figformat=self.figformat, dpi=self.dpi,
+            fig.savefig(os.path.join(self.outDir, plotFile), dpi=self.dpi,
```

Comment:
Looks like matplotlib changed 'figformat' to 'format' (or maybe it always was). 
Can you please put the format choice back in? It should work if you just use 'format = self.figformat')